RevenueCatUI/Data/TestData.swift

```diff
@@ -112,6 +113,10 @@ internal enum TestData {
         callToActionForeground: "#000000"
     )
 
+    #if canImport(SwiftUI) && canImport(UIKit)
+    static let colors: PaywallData.Configuration.Colors = .comine(light: Self.lightColors, dark: Self.darkColors)
+    #endif
```

Comment:
What happens here if these aren't supported? 🤔

Reply:
The views don’t support it either. But when/if we do, the compiler will fail. So it’ll force us to use something else (in practice we’ll expose an NSColor for example).